tests/transport_test.go

```diff
@@ -2,42 +2,52 @@ package tests
 
 import (
 	"fmt"
+	"strconv"
+	"time"
+
+	"net/url"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
-	"github.com/pkg/errors"
-
-	"kubevirt.io/containerized-data-importer/tests/framework"
-	"kubevirt.io/containerized-data-importer/tests/utils"
+	"github.com/golang/glog"
 
 	"k8s.io/api/core/v1"
 	"kubevirt.io/containerized-data-importer/pkg/common"
 	"kubevirt.io/containerized-data-importer/pkg/controller"
+	"kubevirt.io/containerized-data-importer/tests/framework"
+	"kubevirt.io/containerized-data-importer/tests/utils"
 )
 
 var _ = Describe("Transport Tests", func() {
 
 	const (
 		secretPrefix = "transport-e2e-sec"
 		targetFile   = "tinyCore.iso"
+		sizeCheckPod = "size-checker"
 	)
 
-	f, err := framework.NewFramework("", framework.Config{SkipNamespaceCreation: false})
-	handelError(errors.Wrap(err, "error creating test framework"))
+	f := framework.NewFrameworkOrDie("transport", framework.Config{SkipNamespaceCreation: false})
```

Comment:
for SKipNamespaceCreation false should be its default value

Reply:
Correct.  In an earlier comment I believe you asked that I make the field assignment to the default value instead of passing `framework.Config{}`.

Reply:
oh, well I don't know why I suggested that. Sorry. I feel (now I guess!) it's preferable to omit the config if the default values suffice....